modules/rtm/man/bayes_inversion-package.Rd

```diff
@@ -0,0 +1,39 @@
+\name{bayes_inversion-package}
```

Comment:
It is, it was automatically generated by the Rstudio package builder. I can delete it since it doesn't seem to affect the functioning of anything.